pelicun/tests/basic/test_demand_model.py

```diff
@@ -536,21 +582,21 @@ def test__get_required_demand_type(self, assessment_instance):
         ).T.rename_axis(index=['cmp', 'loc', 'dir', 'uid'])
         demand_offset = {'PFA': 0}
         required = _get_required_demand_type(
-            damage_model.ds_model.damage_params, pgb, demand_offset
+            ensure_value(damage_model.ds_model.damage_params), pgb, demand_offset
         )
         expected = defaultdict(
             list,
             {
-                (('PID-1-1', 'PFA-1-1'), 'sqrt(X1^2+X2^2)'): [
+                (('PID-1-1', 'PFA-1-1'), 'sqrt(X1^2+X2^2)'): [  # type: ignore
                     ('testing.component', '1', '1', '1')
                 ]
             },
         )
         assert required == expected
 
-    def test__assemble_required_demand_data(self, assessment_instance):
-
-        # Utility demand case: two demands are required
+    def test__assemble_required_demand_data(
+        self, assessment_instance: Assessment
+    ) -> None:
         damage_model = assessment_instance.damage
```

Comment:
See earlier commend on the "boolean trap". Here we have an example of a case where it wouldn't really be ambiguous, but rather than silencing the warning we can just pass the value as a keyword argument for consistency.